.github/workflows/iroha2-release.yml

```diff
@@ -16,6 +14,10 @@ jobs:
       image: 7272721/i2-ci:nightly
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+      - uses: tj-actions/changed-files@v29.0.2
+        id: changed-files
```

Comment:
I'd be careful. It's not necessarily true that all branches are in a consistent state, because we don't normally update with rebase automatically.

Reply:
Could you rephrase it? I don't think our merge strategy affects the changes detection

Reply:
This check should not fail if the target branch has more recent commits that you don't have (i.e. the update with rebase button exists).

Reply:
Your concern would be that if the diverging branches are identical with respect to `Dockerfile.base` it will detect no changes and fail to update the base image. In that case, there is actually no need to update the base image